.vscode/launch.json

```diff
@@ -0,0 +1,14 @@
+{
+  "configurations": [
+    {
+      "name": "Python Debugger: Remote Attach",
+      "type": "python",
+      "request": "attach",
+      "host": "localhost",
+      "port": 5673,
+      "pathMappings": [
+        { "localRoot": "${workspaceFolder}/app", "remoteRoot": "/app" }
+      ]
+    }
+  ]
+}
```

Comment:
you add [.vscode/launch.json](https://github.com/rharris-nitro/pythonExercise1/pull/9/files#diff-bd5430ee7c51dc892a67b3f2829d1f5b6d223f0fd48b82322cfd45baf9f5e945) but your documentation `Setup local environment` not mention what we need install VSCode